source/features/comment-fields-keyboard-shortcuts.tsx

```diff
@@ -75,6 +80,24 @@ function init(): void {
 					select<HTMLTextAreaElement>('.js-comment-field', lastOwnComment)!.selectionStart = Number.MAX_SAFE_INTEGER;
 				});
 			}
+		} else if (formattingCharacters.includes(event.key)) {
+			const formattingChar = event.key;
+			const [start, end] = [field.selectionStart, field.selectionEnd];
+
+			// If `start` and `end` of selection are the same, then no text is selected
+			// For keys that require selection, if no text is selected, bail immediately
```

Comment:
I don’t want to actually make a text editor in the comment field, I think this feature should just take care of selected text, *if any*, not automatically insert two characters every time you press a key without selection

Reply:
I see it as a small addition to what we can do in a textarea with little to no effort, like not having to press `"`, `` ` `` two times when I can press it only once.

If you say that it might be bit confusing or might looks like we are overdoing things, then I wouldn't mind dropping that specific change.

Reply:
Yeah because in the editor if you delete the character you entered it will also delete the nearby matching character, for example.

Wrapping selected text saves a lot of keystrokes, so it’s worth adding it. Adding a matching character without selection saves one stroke, we can avoid it.

Reply:
Fixed!